### ISSUE TITLE
A distributed brain network predicts general intelligence from resting-state human neuroimaging data

### ISSUE THREAD

**Issue by esoreq** [esoreq/articles#1]
## TL;DR
Individual people differ in their ability to reason, solve problems, think abstractly, plan and learn. The most replicated neural correlate of human intelligence to date is total brain volume; however, this coarse morphometric correlate says little about function. Using a cross-validated predictive framework, we predicted 20% of the variance in general intelligence in the sampled population from their resting-state connectivity matrices.

### Paper Link
https://royalsocietypublishing.org/doi/full/10.1098/rstb.2017.0284

### Author/Institution
Julien Dubois1,4, Paola Galdi5,6, Lynn K. Paul1,3 and Ralph Adolphs1,2,3
California Institute of Technology

## Methods
fMRI
## Comments
